src/Analyzer.JsonRuleEngine.FunctionalTests/RuleParsingTests.cs

```diff
@@ -98,7 +98,7 @@ private static void InequalityValidation(InequalityOperator inequalityOperator,
             Assert.IsTrue(inequalityOperator.Greater);
             Assert.IsFalse(inequalityOperator.OrEquals);
 
-            var parsedDate = DateTime.FromOADate(inequalityOperator.EffectiveValue);
+            var parsedDate = DateTime.FromOADate(inequalityOperator.EffectiveValue).ToUniversalTime();
```

Comment:
@VeraBE No directly released to PSRule, but I found the unit tests were failing in my time zone (+10 GMT).